src/components/Chat/Chat.js

```diff
@@ -17,153 +18,116 @@ import { Streami18n } from '../../Streami18n';
  * It also exposes the withChatContext HOC which you can use to consume the ChatContext
  *
  * @example ../../docs/Chat.md
- * @extends PureComponent
  */
-class Chat extends PureComponent {
-  static propTypes = {
-    /** The StreamChat client object */
-    client: PropTypes.object.isRequired,
-    /**
-     *
-     * Theme could be used for custom styling of the components.
-     *
-     * You can override the classes used in our components under parent theme class.
-     *
-     * e.g. If you want to build a theme where background of message is black
-     *
-     * ```
-     *  <Chat client={client} theme={demo}>
-     *    <Channel>
-     *      <MessageList />
-     *    </Channel>
-     *  </Chat>
-     * ```
-     *
-     * ```scss
-     *  .demo.str-chat {
-     *    .str-chat__message-simple {
-     *      &-text-inner {
-     *        background-color: black;
-     *      }
-     *    }
-     *  }
-     * ```
-     *
-     * Built in available themes:
-     *
-     *  - `messaging light`
-     *  - `messaging dark`
-     *  - `team light`
-     *  - `team dark`
-     *  - `commerce light`
-     *  - `commerce dark`
-     *  - `livestream light`
-     *  - `livestream dark`
-     */
-    theme: PropTypes.string,
-  };
-
-  static defaultProps = {
-    theme: 'messaging light',
-  };
-
-  constructor(props) {
-    super(props);
-
-    this.state = {
-      // currently active channel
-      channel: {},
-      navOpen: true,
-      error: false,
-      mutes: [],
+const Chat = ({ client, theme, i18nInstance, children }) => {
+  const [translators, setTtranslators] = useState({});
```

Comment:
typo in `setTtranslators ` ... extra `t`

Reply:
good catch